demo/DemoArticles/articles.json

```diff
@@ -0,0 +1,14 @@
+[ 
```

Comment:
Now I feel dumb about this. Maybe we should just assume that every article is supported on _both_ MediaWiki action API and the Content Service backends rather doubling the data.